src/llmcompressor/transformers/sparsification/sparse_model.py

```diff
@@ -23,24 +23,19 @@
     resolve_recipe,
 )
 
-__all__ = ["SparseAutoModel", "SparseAutoModelForCausalLM", "get_shared_tokenizer_src"]
+__all__ = [
+    "wrap_hf_model_class",
+    "SparseAutoModel",
+    "SparseAutoModelForCausalLM",
+    "get_shared_tokenizer_src"
+]
 
 
-class SparseAutoModelForCausalLM(AutoModelForCausalLM):
-    """
```

Comment:
This docstring or something like it should be moved to the `from_pretrained` function